packages/core/src/components/results/ResultsExamQuestionManualScore.tsx

```diff
@@ -24,17 +24,14 @@ function ResultsExamQuestionManualScore({ scores, maxScore, displayNumber }: Res
 
   return (
     <ResultsExamQuestionScoresContainer {...containerProps}>
-      {renderNormalizedScores(scores, maxScore)}
+      {!scores?.pregrading && <NoPregrading maxScore={maxScore} />}
```

Comment:
Tää toimii tosiaan nyt koska aina pitää olla pregrading jos on censoring. Toisaalta nyt Results komponentille voi antaa pelkät censoring:it ja komponentti rendaa hassun näköisen lopputuloksen, jossa on eka. `_ / 2 p.`ja sit esim `2 / 2 p.`. 

Onko tämä puhtaasti Results komponentin kannalta toivottavaa vai kannattaisiko vaan iffitellä
`!scores?.pregrading ? <NoPregrading /> : <WithPregrading />` 
(ei toteutusesimerkki 😃 )